src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/ShieldUtilitySpell.java

```diff
@@ -101,16 +116,14 @@ public void appendHoverText(ItemStack pStack, @Nullable Level pLevel, List<Compo
         pTooltipComponents.add(new TextComponent("While equipped in a ").withStyle(ChatFormatting.GRAY)
                                        .append(new TextComponent("utility slot").withStyle(ChatFormatting.DARK_PURPLE))
                                        .append(":"));
-        pTooltipComponents.add(new TextComponent("  - Stores ").withStyle(ChatFormatting.GRAY)
-                                       .append(new TextComponent(totalHits + " charges").withStyle(ChatFormatting.AQUA))
-                                       .append(", each fully negating the damage from an attack"));
-        pTooltipComponents.add(new TextComponent("  - Charges are fully replenished after ").withStyle(ChatFormatting.GRAY)
+        pTooltipComponents.add(new TextComponent("  - Adds ").withStyle(ChatFormatting.GRAY)
+                                       .append(new TextComponent(totalHits + " damage charges").withStyle(ChatFormatting.AQUA))
+                                       .append(", each absorbing the damage from an attack"));
+        pTooltipComponents.add(new TextComponent("  - Charges will begin to replenished after ").withStyle(ChatFormatting.GRAY)
                                        .append(new TextComponent("no charge").withStyle(ChatFormatting.AQUA))
                                        .append(" has been depleted for ")
                                        .append(new TextComponent((recovCooldown / 20) + " seconds").withStyle(ChatFormatting.DARK_GREEN)));
-        pTooltipComponents.add(new TextComponent("Depletes ").withStyle(ChatFormatting.GRAY)
-                                       .append(new TextComponent("1 charge").withStyle(ChatFormatting.AQUA))
-                                       .append(" and costs ")
+        pTooltipComponents.add(new TextComponent("Costs ")
```

Comment:
Add `.withStyle(ChatFormatting.GRAY)` to this `TextComponent`.

Also, I know you changed the wording to say `Adds` instead of `Stores` to imply multiple copies can be stacked, but that alone doesn't make that clear enough IMO. Should add a line to the tooltip above the mana cost line that says something like "Additional copies increase total charges"

Reply:
Fixed